doc/tutorials/crypto.md

```diff
@@ -0,0 +1,152 @@
+We want to protect a password that is contained in an INI-file.
+
+Without encryption, the file could be mounted like:
+
+	sudo kdb mount test.ini user/test ini
+
+We create the password at `user/test/password`:
+
+	kdb set user/test/password 1234
+
+The command above results in the following content of `test.ini`:
+
+	password = 1234
+
+As you can see the password is stored in plain text.
+In this tutorial we demonstrate two different approaches towards confidentiality:
+
+1. with the `fcrypt` plugin, which encrypts the entire INI-file, and
+2. with the `crypto` plugin, which allows the encryption of specific key values only.
+
+We also show how to approach integrity with the signature features of the `fcrypt` plugin.
+
+## Configuration File Encryption/Decryption
+
+The `fcrypt` plugin enables the encryption and decryption of entire configuration files, thus protecting the confidentiality of the configuration keys and values.
+`fcrypt` utilizes GPG for all cryptographic operations.
+The GPG key, which is used for encryption and decryption, is specified in the backend configuration under `encrypt/key`.
+
+	sudo kdb mount test.ini user/test fcrypt "encrypt/key=DDEBEF9EE2DC931701338212DAF635B17F230E8D" ini
+
+As a result the file `test.ini` is encrypted using GnuPG.
+`fcrypt` will call the `gpg2` or `gpg` binary as follows:
+
+	gpg2 -o test.ini -a -r DDEBEF9EE2DC931701338212DAF635B17F230E8D -e test.ini.tmp
+
+Note that `test.ini` can not only be decrypted by Elektra, but it is also possible to decrypt it with GnuPG directly.
+
+## Configuration File Signatures
+
+`fcrypt` also offers the option to sign and verify configuration files, thus protecting the integrity of the configuration values.
+If `sign/key` is specified in the backend configuration, `fcrypt` will forward the key ID to be used for signing the configuration file.
+
+An example backend configuration is given as follows:
+
+	sudo kdb mount test.ini user/test fcrypt "sign/key=DDEBEF9EE2DC931701338212DAF635B17F230E8D" ini
+	
+As a result the file `test.ini` will be signed using GPG.
+`fcrypt` will call the `gpg2` or `gpg` binary as follows:
+
+	gpg2 -o test.ini -a -u DDEBEF9EE2DC931701338212DAF635B17F230E8D -r DDEBEF9EE2DC931701338212DAF635B17F230E8D -s test.ini.tmp
+
+If `test.ini` is modified, all following calls of `kdb get` will fail with an error message stating that the signature of the file could not be verified.
+
+### Combining Signatures and Encryption
+
+The options `sign/key` and `encrypt/key` can be combined together, resulting in configuration files, that are signed and encrypted.
+
+## Configuration Value Encryption/Decryption
+
+The compilation variants of the `crypto` plugin:
+
+1. `crypto_gcrypt`,
+2. `crypto_openssl`, and
+3. `crypto_botan`
+
+provide the option to encrypt and decrypt single configuration values (Keys) in a Keyset.
+`crypto` is using GPG for key-handling.
```

Comment:
While not wrong in any shape or form, I think we should prefer active over passive voice whenever possible:

> `crypto` **uses** GPG for key-handling.

.

Reply:
And to not start with a mono spaced word, you could write:

The plugin `crypto` uses GPG for keys.